### PR TITLE
done tutorial08

### PR DESCRIPTION
**Pull request by a1da4**
遅くなってしまいすみません。
よろしくお願いします。

### CODE REVIEW

File: aida/tutorial08/tutorial08.py [hunk 1]
```diff
@@ -0,0 +1,213 @@
...
+    def forward(self, inputs):
+        """ predict """
+        self._inputs = []
+        self._hiddens = []
+        self._outputs = []
+        hidden = self.init_hidden()
+        for x in inputs:
+            x = np.concatenate((x, hidden), 0)
+            hidden = np.tanh(x@self.weight_h + self.b_h + 1) 
+            output = softmax(hidden@self.weight_o + self.b_o + 1)
```
Comment (kyunashige) [inline aida/tutorial08/tutorial08.py:29]: 👀

File: aida/tutorial08/tutorial08.py [hunk 2]
```diff
@@ -0,0 +1,213 @@
...
+            pred = self.forward(y_test)
+            predictions.append(pred)
+        return predictions
+
+
+def preprocess(file_names):
+    """ obtain X_train, y_train, word_to_id before define model
+
+    :param file_names: list, files[train, test]
+    :param is_train: bool
```
Comment (kyunashige) [inline aida/tutorial08/tutorial08.py:106]: 🤔

File: aida/tutorial08/tutorial08.py [hunk 3]
```diff
@@ -0,0 +1,213 @@
...
+    with open(file_test) as fp:
+        sentences_test = []
+        for line in fp:
+            sentence = []
+            words = line.strip().split()
+            for word in words:
+                if word in word_to_id.keys():
+                    sentence.append(word_to_id[word])
+                else:
+                    sentence.append(word_to_id['<UNK>'])
```
Comment (kyunashige) [inline aida/tutorial08/tutorial08.py:141]:
```suggestion
                    sentence.append(word_to_id[UNK])
```

File: aida/tutorial08/tutorial08.py [hunk 4]
```diff
@@ -0,0 +1,213 @@
...
+    max_epoch = 20
+
+    rnn = RNN(learning_rate, word_to_id, label_to_id, embed_size, hidden_size, output_size)
+    rnn.fit(sentences_train, labels_train, max_epoch)
+    predictions = rnn.predict(sentences_test)
+
+    with open('./tutorial08/my-answer.pos', 'w') as fp:
+        for pred_sent in predictions:
+            labels = decode(pred_sent, id_to_label)
+            fp.write(f'{" ".join(labels)}\n')
```
Comment (kyunashige) [inline aida/tutorial08/tutorial08.py:212]:
```suggestion
            fp.write(" ".join(labels) + '\n')
```

の方が見やすい気がします
`f'UNI:{word}'` の場面では f-strings の方が見やすいかもですが，好みだと思いますし（あとコーディング規約）

ちなみに `print(" ".join(labels), file=fp)` って書くこともできる気がします（遅いかもです）
